source/assets/sass/components/_buttons.scss

```diff
@@ -12,40 +13,37 @@
     radius: functions.sl-px-to-rem(4px);
   }
   padding: var(--gutter-minus) var(--gutter);
-  background: theme.$sl-color--iron;
-  box-shadow: 0 2px 1px rgba(theme.$sl-color--midnight-blue, 0.125);
-  color: color.adjust(theme.$sl-color--hopbush, $lightness: -10%);
+  background: var(--color-button-bg, var(--sl-color-iron));
+  box-shadow: 0 2px 1px var(--shadow);
+  color: var(--color-button-text, var(--highlight));
   cursor: pointer;
 
   &:hover,
   &:focus {
-    color: color.adjust(theme.$sl-color--bouquet, $lightness: -10%);
+    --color-button-text: var(--action);
   }
 
   &:active {
-    color: color.adjust(theme.$sl-color--venus, $lightness: -10%);
+    --color-button-text: color.adjust(brand.$sl-color--venus, $lightness: -10%);
   }
 
   &--primary {
-    background-color: color.adjust(theme.$sl-color--hopbush, $lightness: -10%);
+    --color-button-bg: var(--highlight);
 
     &,
     &:hover,
     &:focus,
     &:active {
-      color: theme.$sl-color--white;
+      --color-button-text: var(--sl-color--white);
     }
 
     &:hover,
     &:focus {
-      background-color: color.adjust(
-        theme.$sl-color--bouquet,
-        $lightness: -10%
-      );
+      background-color: var(--action);
```

Comment:
For the text color, we're changing the custom prop instead. Should we be setting `--color-button-bg` rather than `background-color` on these (see also `:active`).

Reply:
@mirisuzanne I had to give `a` tag currently in `visual-design/_theme` a `--color-action-bg` custom property to make this work on `:hover` and `:focus`.  Do you see a more elegant solution?